### PR TITLE
feat: Video Stream API

### PR DESCRIPTION
**Pull request by jsoizo**
This PR adds support for Video Stream API 
https://api.cloudflare.com/#stream-videos-properties

### CODE REVIEW

File: lib/resources/StreamVideos.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+ * StreamVideos represents the /accout/:id/stream API endpoint.
+ *
+ * @class StreamVideos
+ * @hideconstructor
+ * @extends Resource
+ */
+module.exports = auto(
+  prototypal({
+    extends: Resource,
+    path: 'accounts',
```
Comment (terinjokes) [inline lib/resources/StreamVideos.js:26]: This should probably be `accountd/:accountId/stream` and the other paths updated. (I suspect the "basic" paths will just work after that)

File: index.js [hunk 2]
```diff
@@ -28,6 +28,7 @@ const resources = {
   zoneWorkersScript: require('./lib/resources/ZoneWorkersScript'),
   zoneWorkersRoutes: require('./lib/resources/ZoneWorkersRoutes'),
   user: require('./lib/resources/User'),
+  streamVideos: require('./lib/resources/StreamVideos'),
```
Comment (terinjokes) [inline index.js:31]: I think this can just be "stream".

**Comment by jsoizo** [comment]
@terinjokes Sorry, I will fix it soon.

**Comment by terinjokes** [comment]
Merged as 2079b457e7ceae23e52e1bfa4d426a5a53ceb81b.

**Pull request closed without merging**